docs/Migrating to Newer SDL Versions/Migrating SDL Core 6.1 to 7.0/index.md

```diff
@@ -0,0 +1,252 @@
+
+```
+<function name="SubtleAlert" messagetype="request">
+    <description>Request from SDL to show a subtle alert message on the display.</description>
+    <param name="alertStrings" type="Common.TextFieldStruct" mandatory="true" array="true" minsize="0" maxsize="2">
+        <description>Array of lines of alert text fields. See TextFieldStruct. Uses subtleAlertText1, subtleAlertText2.</description>
+    </param>
+    <param name="alertIcon" type="Common.Image" mandatory="false">
+        <description>
+            Image to be displayed for the corresponding alert. See Image. 
+            If omitted, no (or the default if applicable) icon should be displayed.
+        </description>
+    </param>
+    <param name="duration" type="Integer" mandatory="false" minvalue="3000" maxvalue="10000">
+        <description>Timeout in milliseconds.</description>
+    </param>
+    <param name="softButtons" type="Common.SoftButton" mandatory="false" minsize="0" maxsize="2" array="true">
+        <description>App defined SoftButtons</description>
+    </param>
+    <param name="alertType" type="Common.AlertType" mandatory="true">
+        <description>Defines if only UI or BOTH portions of the Alert request are being sent to HMI Side</description>
+    </param>
+    <param name="appID" type="Integer" mandatory="true">
+        <description>ID of application requested this RPC.</description>
+    </param>
+    <param name="cancelID" type="Integer" mandatory="false">
+        <description>
+            An ID for this specific alert to allow cancellation through the `CancelInteraction` RPC.
+        </description>
+    </param>
+</function>
+
+<function name="SubtleAlert" messagetype="response">
+    <param name="tryAgainTime" type="Integer" mandatory="false" minvalue="0" maxvalue="2000000000">
+        <description>Amount of time (in milliseconds) that SDL must wait before resending an alert. Must be provided if another system event or overlay currently has a higher priority than this alert.</description>
+    </param>
+</function>
+
+<function name="OnSubtleAlertPressed" messagetype="notification">
+    <description>
+    Sent when the alert itself is touched (outside of a soft button). Touching (or otherwise selecting) the alert should open the app before sending this notification.
+    </description>
+    <param name="appID" type="Integer" mandatory="true">
+        <description>ID of application that is related to this RPC.</description>
+    </param>
+</function>
+```
+
+### Webengine Projection Support
+
+In SDL Core 6.1 Webengine apps was added as a way to connect an SDL Connected app to the head unit using a headless web browser on the head unit. This feature was expanded in SDL Core 7.0 With the addition of a new HMI_TYPE and template layout `WEB_VIEW`. Apps that use this HMI_TYPE and layout can present their own non-templated layout using HTML and javascript.
```

Comment:
WEB_VIEW only applies to web engine, but it's not too related other than that. I would maybe remove some of the prelude. Here are some things I think could be added:

A new `WEB_VIEW` appHMIType and template layout was added which will allow apps to render a template independent view in a browser environment with javascript and HTML. 
This appHMIType must be explicitly specified in an app's policy table entry for SDL Core to allow it to be used.